tests/test_formatter.py

```diff
@@ -964,6 +970,30 @@ def test_no_inline_comments_stay_untouched(self):
         formatter = setup_formatter(snakecode)
         assert formatter.get_formatted() == snakecode
 
+    def test_snakecode_after_indented_comment_does_not_get_unindented(self):
+        """https://github.com/snakemake/snakefmt/issues/159#issue-1441174995"""
+        snakecode = (
+            'if config.get("s3_dst"):\n\n'
+            f'{TAB * 1}include: "workflow/rule1.smk"\n'
+            f'{TAB * 1}include: "workflow/rule2.smk"\n'
+            f"{TAB * 1}# a comment\n"
```

Comment:
Shouldn't there be a new line between these two lines?

Reply:
I don't think so. our policy has been that if you have a series of consecutive keywords (here, `include`s), *possibly interspersed with comments* you don't space any of the lines out

Reply:
If you replace the final `include` with, say, `shell`, you will get that newline before the first comment

Reply:
Ahhh very nice!! Too smart for me